docs/source/usage/plugins/isaac.rst

```diff
@@ -74,6 +74,20 @@ Beside the functor chains the client allows to setup the weights per source (val
 Furthermore interpolation can be activated.
 However this is quite slow and most of the time not needed for non-iso-surface rendering.
 
+Memory Complexity
+^^^^^^^^^^^^^^^^^
+
+Accelerator
+"""""""""""
+
+locally, a framebuffer with full resolution and 3 color channels and 4 byte is allocated.
+For each ``FieldTmp`` derived field and ``FieldJ`` a copy is allocated, depending on the input in the ``.param`` file.
```

Comment:
Is there a reason, you don't point to the exact `.param` file, namely the `isaac.param` file?